### PR TITLE
Update pt.json

### PR DESCRIPTION
**Pull request by bernardoamorim7**
From line 404 to line 463.

Other portuguese speakers might want to give a look too.

### CODE REVIEW

File: src/docs/src/translation/pt.json [hunk 1]
```diff
@@ -401,69 +401,64 @@
...
-  "All radio inputs with the same name work together and only one of them can be open at a time. If you have more than one set of accordion items on a page, use different names for the radio inputs on each set.": "All radio inputs with the same name work together and only one of them can be open at a time. If you have more than one set of accordion items on a page, use different names for the radio inputs on each set.",
-  "Join": "Join",
-  "Join is a container for grouping multiple items, it can be used to group buttons, inputs, or any other element. Join applies border radius to the first and last item. Join can be used to create a horizontal or vertical list of items.": "Join is a container for grouping multiple items, it can be used to group buttons, inputs, or any other element. Join applies border radius to the first and last item. Join can be used to create a horizontal or vertical list of items.",
-  "Loading": "Loading",
-  "Loading shows an animation to indicate that something is loading.": "Loading shows an animation to indicate that something is loading.",
-  "Browser mockup": "Browser mockup",
```
Comment (saadeghi) [inline src/docs/src/translation/pt.json:414]: Also

File: src/docs/src/translation/pt.json [hunk 2]
```diff
@@ -401,69 +401,64 @@
...
-  "Browser mockup shows a box that looks like a browser window.": "Browser mockup shows a box that looks like a browser window.",
-  "Browser": "Browser",
```
Comment (saadeghi) [inline src/docs/src/translation/pt.json:416]: Also

**Review by saadeghi — CHANGES_REQUESTED** [review]

**Comment by bernardoamorim7** [comment]
I apologize, didn't even see that when submitting the PR.
Should be fixed now, plus other mistakes I found on the translated part.

**Comment by saadeghi** [comment]
Thanks!

**Pull request merged**